oaknational/Oak-Web-Application
# feat: a11y opens in a new tab [LESQ-895]

## Description

Music year: 1978

- Update footer links which open in new tab to have an aria-label saying as such


## How to test

1. Go to https://deploy-preview-2540--oak-web-application.netlify.thenational.academy
2. Check the attributes for the links in the footer that open in a new tab
3. You should see `(opens in a new tab)` after the link name

## Screenshots

How it used to look (delete if n/a):

<img width="300" alt="Screenshot 2024-06-26 at 15 33 23" src="https://github.com/oaknational/Oak-Web-Application/assets/45038878/c246f98f-6c78-4606-9758-4b165e2b998d">


How it should now look:

<img width="300" alt="Screenshot 2024-06-26 at 15 32 53" src="https://github.com/oaknational/Oak-Web-Application/assets/45038878/28431aa0-1ca5-45be-b2c6-1707b5fa72ef">


## Checklist

- [x] All footer links that open in a new tab announce to screen reader users that this link will open in a new tab (much like the equivalent in the hamburger menu do)
